p8e-api/src/main/kotlin/io/provenance/engine/batch/MailboxReaper.kt

```diff
@@ -256,22 +220,48 @@ class MailboxReaper(
      */
     @Scheduled(initialDelayString = "\${reaper.inbox.delay}", fixedDelayString = "\${reaper.inbox.interval}")
     fun pollInbound() {
+        MDC.clear()
+
         log.debug("Polling mailbox reaper inbound")
 
+        // TODO catch status runtime exceptions and don't rethrow
```

Comment:
is this todo taken care of now by the try/catch below? Or is this about catching things other than UNAVAILABLE?